pygin/pygin/src/panel.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
```

Comment:
Please use hpp extension.

Reply:
Fixed in 5ef1f612ff8787268b8d48658e3d1e8cad9b7272.